### DISCUSSION TITLE
get forward_from in Bot.update

### DISCUSSION THREAD

**Discussion by hkpmatt** [python-telegram-bot/python-telegram-bot#3845] [Feature Suggestions]
<img width="653" alt="image" src="https://github.com/python-telegram-bot/python-telegram-bot/assets/70888649/4a0913a1-ceff-43d1-9e7f-9ebd4b239544">

**Comment by Bibo-Joshi** [comment]
Hi. Please ask a proper question. See [This article](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Ask-Right).

**Comment by hkpmatt** [comment]
![image](https://github.com/python-telegram-bot/python-telegram-bot/assets/70888649/45b6b04c-0330-4b91-9588-8a239bd5440e)
(this is account hidden by user)

For user that didn't hide their account for forward message.

Update will get
**`Message(channel_chat_created=False, chat=Chat(api_kwargs={'all_members_are_administrators': True}, id=chatid, title='', type=<ChatType.GROUP>), date=datetime.datetime(2023, 8, 15, 6, 9, 36, tzinfo=<UTC>), delete_chat_photo=False, from_user=User(first_name='name', id=chatid, is_bot=False, language_code='en', username='username'), group_chat_created=False, message_id=460, reply_to_message=Message(channel_chat_created=False, chat=Chat(api_kwargs={'all_members_are_administrators': True}, id=chatid, title='', type=<ChatType.GROUP>), date=datetime.datetime(2023, 8, 14, 9, 17, 3, tzinfo=<UTC>), delete_chat_photo=False, forward_date=datetime.datetime(2023, 8, 14, 9, 17, 2, tzinfo=<UTC>), forward_from=User(first_name='???', id=chatid, is_bot=False, language_code='zh-hans', username='usernaame'), from_user=User(first_name='username', id=chatid, is_bot=True, username='username'), group_chat_created=False, message_id=312, supergroup_chat_created=False, text='Hi'), supergroup_chat_created=False, text='bye')`**

We can see there is **`forward_from=User(first_name='???', id=chatid, is_bot=False, language_code='zh-hans', username='usernaame')`**
which can get chat id from the originally sender of the message when you reply to the the forwarded message.
Therefore, I suggest adding an option to get chat_id from forward_from to get the originally sender id combine with forward_message to achieve reply to the original sender while replying to the forwarded message from the bot.

**Comment by harshil21** [comment]
Closing in favor of #3843
